src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -494,15 +494,15 @@ class Variable
                 state.SendBackspace()
                 ' This completion is hard-selected because the suggestion mode never triggers on backspace
                 ' See issue https://github.com/dotnet/roslyn/issues/15302
-                Await state.AssertSelectedCompletionItem(displayText:="as", isHardSelected:=True)
+                Await state.AssertSelectedCompletionItem(displayText:="as", isHardSelected:=False)
```

Comment:
This is a test @jcouv wrote. But couldn't you be writing a parenthesized expression with an as clause?